src/averageAges.js

```diff
@@ -37,7 +46,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const namesMentionedAsMother = people.map(p => p.mother);
+  const filteredWomen = people
+    .filter(p => p.sex === 'f'
```

Comment:
```suggestion
    .filter(person => p.sex === 'f'
```
do not use short forms